src/software/ai/hl/stp/evaluation/possession.cpp

```diff
@@ -69,4 +71,29 @@ namespace Evaluation
 
         return false;
     }
+
+    bool teamPassInProgress(const Ball &ball, const Team &team)
```

Comment:
Doesn't look like this function is tested at all? Could we get the person who wrote it to write a few tests? If it's a hassle, no worries, I can write the tests, just let me know.

Reply:
I created an issue (#919) to unit test this